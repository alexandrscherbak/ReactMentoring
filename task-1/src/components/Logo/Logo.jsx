import React from 'react';

const Logo = ({count}) => (
	<div>
		<span>netflixroulette</span>

		 <style jsx>
			{`
				span {
					color: #99444f;
				}
			`}
		</style> 
	</div>
);

export default Logo;
