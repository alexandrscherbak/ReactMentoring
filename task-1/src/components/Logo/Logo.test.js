import React from 'react';
import Logo from './Logo.jsx';
import renderer from 'react-test-renderer';

describe('Logo', () => {
	test('snapshot', () => {
		const component = renderer.create(
			<Logo/>,
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
