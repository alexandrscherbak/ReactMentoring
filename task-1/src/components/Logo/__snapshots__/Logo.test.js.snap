// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo snapshot 1`] = `
<div
  className="jsx-2852891300"
>
  <span
    className="jsx-2852891300"
  >
    netflixroulette
  </span>
</div>
`;
